kernel/src/main/scala/org/typelevel/sbt/TypelevelKernelPlugin.scala

```diff
@@ -29,7 +29,8 @@ object TypelevelKernelPlugin extends AutoPlugin {
   override def trigger = allRequirements
 
   object autoImport {
-    lazy val CompileTime: Configuration = config("compile-time").hide
+    @deprecated("use `Provided` instead", "0.6.1")
+    lazy val CompileTime: Configuration = Provided
```

Comment:
Since we're deprecating it anyway I wonder if we should leave the old definition. Also I wonder if anyone could have been using it like `"compile-time"` instead of `CompileTime`.